docs_app/content/guide/observable.md

```diff
@@ -28,21 +28,21 @@ To invoke the Observable and see these values, we need to *subscribe* to it:
 ```ts
 import { Observable } from 'rxjs';
 
-const observable = Observable.create(function (observer) {
-  observer.next(1);
-  observer.next(2);
-  observer.next(3);
+const observable = new Observable(subscriber => {
+  subscriber.next(1);
+  subscriber.next(2);
+  subscriber.next(3);
   setTimeout(() => {
-    observer.next(4);
-    observer.complete();
+    subscriber.next(4);
+    subscriber.complete();
   }, 1000);
 });
 
 console.log('just before subscribe');
 observable.subscribe({
-  next: x => console.log('got value ' + x),
-  error: err => console.error('something wrong occurred: ' + err),
-  complete: () => console.log('done'),
+  next(x) { console.log('got value ' + x); },
```

Comment:
what do you think about changing to using template strings?

Reply:
It makes sense to use it. Will make the example a bit more clear.